File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public static char[] ch = {'a','b','c',
             'd','e','f',
             'g','h','i',
             'j','k','l',
             'm','n','o',
             'p','q','r',
             's','t','u',
             'v','w',
             'x','y','z'};
    public boolean checkIfPangram(String s) {
        for(int i=0;i<ch.length;i++){
                if (s.indexOf(ch[i]) == -1) return false;
        }
        return true;
    }
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, s: str) -> bool:
        if len(s)<26:
            return False
        l = ['a','b','c',
             'd','e','f',
             'g','h','i',
             'j','k','l',
             'm','n','o',
             'p','q','r',
             's','t','u',
             'v','w',
             'x','y','z']
        for i in l:
            if i not in s:
                return False
        return True